### PR TITLE
Update what-is-gitops.mdx

### PR DESCRIPTION
**Pull request by deepeshaburse**
Signed-off-by: Deepesha Burse <87636253+deepeshaburse@users.noreply.github.com>

Update for the GitOps resource article.

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/collections/resources/articles/gitops/what-is-gitops.mdx [hunk 1]
```diff
@@ -18,38 +18,63 @@ import { ResourcesWrapper } from "../../Resources.style.js";
...
+ </p>
+ 
+ <p> GitOps also improves a system's ability to recover from failure because it's simple to roll back an unsuccessful change or restore the entire system from the repository.
+</p>
 
-If you are new to GitOps, it builds and iterates on ideas drawn from DevOps and Infrastructure as Code that started with Martin Fowler’s comprehensive Continuous Integration overview and provides the freedom to choose the tools that you need for your specific use cases.
+### 3) Automated Change Delivery
+ <p> Only automation allows us to apply modifications made to the blueprint to systems already in operation. Delivery of changes is entirely automatic. GitOps doesn't allow manual editing. Because standard workflows only need Git, which is such a well-known platform, automation enables changes to be delivered through simpler for developers to use workflows. Additionally, automation standardizes your delivery processes, improving the predictability and consistency of system operations.
```
Comment (legendarykamal) [inline src/collections/resources/articles/gitops/what-is-gitops.mdx:44]: Git or github ? 
as git is not a platform it is  a DevOps tool used for source code management. Automation is done through github

**Comment by deepeshaburse** [comment]
> Rest looks good :)

Thank you Kamal, I have made the changes :)

**Comment by leecalcote** [comment]
The build seems to be failing....

**Comment by leecalcote** [comment]
Kicking off a new build. If it fails, @deepeshaburse, you'll need to have another look and confirm that this branch is building on your local machine.

**Comment by deepeshaburse** [comment]
> Kicking off a new build. If it fails, @deepeshaburse, you'll need to have another look and confirm that this branch is building on your local machine.

Okay 👍🏼

**Comment by leecalcote** [comment]
Still failing, @deepeshaburse

**Pull request merged**